pkgs/top-level/all-packages.nix

```diff
@@ -83,6 +83,11 @@ let
   platform = if platform_ != null then platform_
     else config.platform or platformAuto;
 
+  topLevelArguments = {
+    inherit system bootStdenv noSysDirs gccWithCC gccWithProfiling config
+      crossSystem platform lib;
+  };
```

Comment:
Aliasing these arguments seem a bit superfluous if they can be passed to `import ./stdenv {}` directly. A later change might invalidate my thinking.